src/engine/ruleset/decoders/redis/redis.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: decoder/redis/0
+
+metadata:
+  description: Decoder for Linux redis logs
+  references:
+    - https://docs.elastic.co/en/integrations/redis
```

Comment:
Nitpick (Documentation): could we add official redis documentation instead of elastic?

Reply:
From what I saw in the other decoders they put references to elastic. what do you think ? @cborla